src/share/poudriere/jail.sh

```diff
@@ -64,6 +64,10 @@ Options:
                      building for a different TARGET ARCH than the host.
                      Only applies if TARGET_ARCH and HOST_ARCH are different.
                      Will only be used if -m is svn*.
+Options for -d:
+    -C clean      -- Clean remaining data existing in poudriere data folder.
+                     See poudriere(8) for more details. Can be one of:
+                       all, logs, packages, wrkdirs
```

Comment:
Double check these 3 lines to make sure there's no TABS and only spaces. It looks weird here; the usage is only using spaces. In vim you can do `:set list`